vicodyn/include/cocaine/vicodyn/peer.hpp

```diff
@@ -65,6 +71,7 @@ class peer_t : public std::enable_shared_from_this<peer_t> {
     std::unique_ptr<logging::logger_t> logger_;
     synchronized<std::shared_ptr<cocaine::session_t>> session_;
     bool connecting_{false};
+    std::atomic<std::chrono::milliseconds::rep> ban_until_ms_{0};
```

Comment:
Let's store `std::chrono::steady_clock::time_point` when the ban should be removed.

Reply:
In that case we'll have to use mutex for synchronization. Atomic is more efficient.